React/Modules/RCTI18nUtil.h

```diff
@@ -18,11 +18,22 @@
 + (instancetype)sharedInstance;
 
 - (BOOL)isRTL;
-- (BOOL)isRTLAllowed;
-- (void)allowRTL:(BOOL)value;
-- (BOOL)isRTLForced;
-- (void)forceRTL:(BOOL)value;
-- (BOOL)doLeftAndRightSwapInRTL;
-- (void)swapLeftAndRightInRTL:(BOOL)value;
+
+/**
+ * Should be used very early during app start up
+ * Before the bridge is initialized
+ * @return whether the app allows RTL layout, default is true
+ */
+@property(atomic, readwrite, setter=allowRTL:) bool isRTLAllowed;
+
+/**
+ * Could be used to test RTL layout with English
+ * Used for development and testing purpose
+ */
+@property(atomic, readwrite, setter=forceRTL:) bool isRTLForced;
+
+@property(atomic, readwrite, setter=swapLeftAndRightInRTL:) bool doLeftAndRightSwapInRTL;
```

Comment:
For all the changes in this file, can you surround them with a `TODO: ` and tracking bug number to bring it back upstream to FB. We want all diffs between our fork and facebook's to be tracked to make merges easier

Reply:
I'll hold off on that in case the upstream PR goes in. But thanks for commenting. Are those issue #'s all tracked via the github issue tracker?

Reply:
Yea at this point we should just track them on GitHub instead of ISS. You might need to open a bug on it

Reply:
I'll give my usual pitch about about the fact that we've been trying to use work items to track reconciling divergences for years but the existence of the work items has never actually provided any practical value that I've seen. I personally would vote against creating a work item and instead would vote for spending our energy adding comments in the code itself about why this is different from upstream with the target audience being a well-meaning engineer in the future trying to do a merge or remove a diff.

Reply:
I'd absolutely like a meaningful comment. We do want this upstream so tracking that work with a bug as well seems noble here.